stories/covid-vax/covid_vax.ipynb

```diff
@@ -0,0 +1,1597 @@
+{
```

Comment:
<p>Relates to the comment above - might need to reword "date a group was fully vaccinated" if there is a lingering percent of each group that has been offered the vaccine but not yet had it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>